docs/database/cosmosdb.md

```diff
@@ -14,6 +14,14 @@ linkTitle: Cosmos DB
     - [Metric: `db.client.response.returned_rows`](#metric-dbclientresponsereturned_rows)
     - [Metric: `db.client.cosmosdb.operation.request_charge`](#metric-dbclientcosmosdboperationrequest_charge)
     - [Metric: `db.client.cosmosdb.active_instance.count`](#metric-dbclientcosmosdbactive_instancecount)
+  - [Network Level Metrics](#network-level-metrics)
+    - [Metric: `db.client.cosmosdb.request.duration`](#metric-dbclientcosmosdbrequestduration)
+    - [Metric: `db.client.cosmosdb.request.body.size`](#metric-dbclientcosmosdbrequestbodysize)
+    - [Metric: `db.client.cosmosdb.response.body.size`](#metric-dbclientcosmosdbresponsebodysize)
+    - [Metric: `db.client.cosmosdb.request.channel_aquisition.duration`](#metric-dbclientcosmosdbrequestchannel_aquisitionduration)
```

Comment:
Please remove this metric from semantic profile. It will go away soon anyway (ThinClient) - so, fine to emit it for direct mode - but adding it to semantic profile to then removing it in a couple of months is bad.